ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/TransactionServiceIT.scala

```diff
@@ -1652,7 +1653,7 @@ object TransactionServiceIT {
         t.copy(
           commandId = "commandId",
           offset = "offset",
-          eventsById = t.eventsById.mapValues(_.modifyWitnessParties(_.sorted)),
+          eventsById = t.eventsById.view.mapValues(_.modifyWitnessParties(_.sorted)).toMap,
```

Comment:
```suggestion
          eventsById = t.eventsById.mapValues(_.modifyWitnessParties(_.sorted)),
```
Can we simply apply the transformation without laziness? If your concern is the recomputing issue of `mapValues`, I believe it should be fixed in 2.13 (if I'm not mistaken).

Reply:
`mapValues` is also deprecated https://www.scala-lang.org/api/current/scala/collection/immutable/Map.html#mapValues[W](f:V=%3EW):scala.collection.MapView[K,W]

Reply:
in other words, the "fix" is to make you aware that `mapValues` will return a view by throwing a deprecation warning. They don’t yet have a strict version afaik.

Reply:
Thanks!